app/routes.php

```diff
@@ -24,14 +26,25 @@
 
 Route::get('logout','HomeController@logout');
 
+Route::post('/signup','HomeController@signUp');
+
+Route::get('hello', 'HomeController@showWelcome');
```

Comment:
its for something i was working on in trying to make the game into an RPG format its not important and is not connected to other parts in the game, but if need be it can be deleted